### PR TITLE
chore: use latest beta (aka zzz-beta)

### PR DESCRIPTION
**Pull request by sobolk**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] Any CDK or CloudFormation parameter changes are called out explicitly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by phani-srikar** [comment]
Can you please run e2e on this? I'm seeing another failure with using the latest `zzz-beta` tag. Refer [here](https://app.circleci.com/pipelines/github/aws-amplify/amplify-category-api/4818/workflows/a169121a-5f27-4ea1-a562-e111004f1d59/jobs/93915/steps)

**Comment by sobolk** [comment]
> Can you please run e2e on this? I'm seeing another failure with using the latest `zzz-beta` tag. Refer [here](https://app.circleci.com/pipelines/github/aws-amplify/amplify-category-api/4818/workflows/a169121a-5f27-4ea1-a562-e111004f1d59/jobs/93915/steps)

yeah. I keep doing this.

**Comment by sobolk** [comment]
https://app.circleci.com/pipelines/github/aws-amplify/amplify-category-api?branch=run-e2e%2Fsobkamil%2Fsobolk-patch-1

File: packages/amplify-util-mock/src/func/index.ts [hunk 1]
```diff
@@ -45,7 +47,7 @@ export async function start(context: $TSContext) {
   const invoker = await getInvoker(context, { resourceName, handler: lambdaConfig.handler, envVars: lambdaConfig.environment });
   context.print.blue('Starting execution...');
   try {
-    const result = await timeConstrainedInvoker(invoker({ event }), context.input.options);
+    const result = await timeConstrainedInvoker(invoker({ event }), context.input.options as $TSAny);
```
Comment (sobolk) [inline packages/amplify-util-mock/src/func/index.ts:50]: this was typed as `$TSAny` before on the CLI side, so I'm casting this for simplicty, the target type is anyway one optional property.
@sdstolworthy we should dry-run context changes in data repo through tag release going forward.

**Review by phani-srikar — DISMISSED** [review]

**Review by alharris-at — APPROVED** [review]

**Review by phani-srikar — APPROVED** [review]

**Pull request merged**